contracts/script/multisigTransactionProposals/safeSDK/upgradeProxy.ts

```diff
@@ -32,60 +41,120 @@ async function main() {
 
     const chainId = await ethAdapter.getChainId();
     const safeService = new SafeApiKit({ chainId });
-    const safeAddress = getEnvVar("SAFE_MULTISIG_ADDRESS");
-    validateEthereumAddress(safeAddress);
-    const safeSdk = await Safe.create({ ethAdapter, safeAddress });
+    validateEthereumAddress(upgradeData.safeAddress);
+    const safeSdk = await Safe.create({ ethAdapter, safeAddress: upgradeData.safeAddress });
     const orchestratorSignerAddress = await orchestratorSigner.getAddress();
 
-    await proposeUpgradeTransaction(safeSdk, safeService, orchestratorSignerAddress, safeAddress, upgradeData);
+    await proposeUpgradeTransaction(safeSdk, safeService, orchestratorSignerAddress, upgradeData);
 
     console.log(
-      `The other owners of the Safe Multisig wallet need to sign the transaction via the Safe UI https://app.safe.global/transactions/queue?safe=sep:${safeAddress}`,
+      `The other owners of the Safe Multisig wallet need to sign the transaction via the Safe UI https://app.safe.global/transactions/queue?safe=sep:${upgradeData.safeAddress}`,
     );
   } catch (error) {
     throw new Error("An error occurred: " + error);
   }
 }
 
+export function processRustCommandLineArguments(args: string[]): UpgradeData {
```

Comment:
Can we use a command line parser lib so we can outsource parsing of the command  line arguments. I don't know the node ecosystem well. I know about https://github.com/yargs/yargs, maybe @Ayiga has a preferred one?

Reply:
no longer required as per our convo